panic: runtime error: index out of range [0] with length 0

## Expected behavior

list lifecycle configuration rules set on a bucket

## Actual behavior

```shell
$ mc ilm ls --transition s3/my-bucket
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/minio/mc/cmd.ilmListMessage.String(0x10f99d2, 0x7, 0x7fffeea93ec5, 0x1f, 0xc000283a00, 0xc0003f8700, 0x203000, 0x1167f70)
        github.com/minio/mc/cmd/ilm-ls.go:125 +0x62f
github.com/minio/mc/cmd.printMsg(0x1271e98, 0xc00009b170)
        github.com/minio/mc/cmd/print.go:38 +0x25f
github.com/minio/mc/cmd.mainILMList(0xc000283a00, 0x0, 0x0)
        github.com/minio/mc/cmd/ilm-ls.go:273 +0x305
github.com/minio/cli.HandleAction(0xf724e0, 0x1167cb0, 0xc000283a00, 0xc00059d400, 0x0)
        github.com/minio/cli@v1.22.0/app.go:498 +0x82
github.com/minio/cli.Command.Run(0x10f40a6, 0x2, 0x0, 0x0, 0x0, 0x0, 0x0, 0x112fdc0, 0x33, 0x0, ...)
        github.com/minio/cli@v1.22.0/command.go:225 +0x9bb
github.com/minio/cli.(*App).RunAsSubcommand(0xc0002824e0, 0xc0002821a0, 0x0, 0x0)
        github.com/minio/cli@v1.22.0/app.go:387 +0x894
github.com/minio/cli.Command.startApp(0x10f496a, 0x3, 0x0, 0x0, 0x0, 0x0, 0x0, 0x110b7f5, 0x17, 0x0, ...)
        github.com/minio/cli@v1.22.0/command.go:313 +0x7bf
github.com/minio/cli.Command.Run(0x10f496a, 0x3, 0x0, 0x0, 0x0, 0x0, 0x0, 0x110b7f5, 0x17, 0x0, ...)
        github.com/minio/cli@v1.22.0/command.go:111 +0x11f0
github.com/minio/cli.(*App).Run(0xc00017c000, 0xc0000300a0, 0x5, 0x5, 0x0, 0x0)
        github.com/minio/cli@v1.22.0/app.go:261 +0x768
github.com/minio/mc/cmd.Main(0xc0000300a0, 0x5, 0x5)
        github.com/minio/mc/cmd/main.go:124 +0x33a
main.main()
        github.com/minio/mc/main.go:29 +0x45
```

## Steps to reproduce the behavior

Just run the following command for an existing bucket on DigitalOcean:

```shell
$ mc ilm ls --transition s3/my-bucket
```

## mc --version
mc version RELEASE.2021-07-27T06-46-19Z

## System information
Ubuntu 18.04.5 LTS

--- a/cmd/ilm-ls.go
+++ b/cmd/ilm-ls.go
@@ -116,8 +116,10 @@ func (i ilmListMessage) String() string {
 	tbl = newPrettyTable(tableSeperator, fields...)
 	tblContents = getILMHeader(&tbl, alignedHdrLabels...)
 
-	// Reuse the fields
-	fields = nil
+	// If no data to show at all, quit here
+	if len(cellDataNoTags) == 0 && len(cellDataWithTags) == 0 {
+		return tblContents
+	}
 
 	// The data table
 	var tblRowField *[]string
@@ -126,6 +128,9 @@ func (i ilmListMessage) String() string {
 	} else {
 		tblRowField = &cellDataNoTags[0]
 	}
+
+	// Reuse the fields
+	fields = nil
 
 	for _, hdr := range *tblRowField {
 		fields = append(fields, Field{ilmThemeRow, len(hdr)})

